.github/workflows/freebsd.yml

```diff
@@ -5,22 +5,39 @@ on:
     paths-ignore:
     - '.github/workflows/**'
     - '!.github/workflows/freebsd.yml'
+    - 'doc/**'
```

Comment:
libva is pretty small project. I do not see the point to invest in such a list.

Reply:
Updated the PR dropping the `paths-ignore` section all together.